index.rst

```diff
@@ -5,59 +5,108 @@ Technote Style and Writing Guide
 Abstract
 ========
 
-.. abstract::
+Quick reference for writing SITCOM technotes. Each section in this technote is intended to reflect the actual suggested structure. It is advisable to maintain the main headers homogeneous across technotes, and structure the report in subheaders as needed by the author. 
 
-   A guide to on how to style and write a Technote
+Technote abstracts should include a short 2-3 sentence explanation of the goal of the technote. Ideally, it should also include a 1-2 sentence summary of the main conclusions.
 
-Introduction
-================
+*Example: We verify that the pointing accuracy of the system is compliant with the requirement of 5 arcsec RMS. We do this by analyzing star tracker data during a soak test mimicking real scheduled observations. Our conclusions are that the pointing performance is compliant with the requirement, except at low (<30 deg.) elevation measurements.* 
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+*Example: We explore the problems found on 2024-02-01 on the azimuth drives to diagnose the issue. We study what TMA settings lead to the drives going to fault. Currently not correlation of velocity, acceleration, jerk or initial/final positions are found*.
 
-Main part
-=========
 
-Technote Style Guide
---------------------
-Text Parts
-^^^^^^^^^^
-Reasoning:
-Using the same structure in all technotes makes it easy to jump in to the needed information
+Introduction
+============
 
-A document should be structured in the following parts
+This is a short reference to provide guidance and support for creating technotes related to SITCOM activities. For details on how to start, configure and edit a technote, see the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_. This includes adding yourself as author in the author database file, in case you are not there.
 
-- Introduction
-- Main Part
-- Summary and Conclusion
+The introduction should expand the details and context in which the technote is written, and why its goals are pursued. In the case of a very straightforward measurement, or self-exaplanatory, it could be merged with the abstract. Ideally these would be exceptions.
```

Comment:
For readability, consider dropping the "Ideally these would be exceptions.". I think the prior sentence makes the point you want.

Reply:
Done.